frontend/public/components/catalog/_catalog.scss

```diff
@@ -20,6 +16,15 @@ $catalog-tile-width: $co-m-catalog-tile-width;
   font-size: ($font-size-base - 1);
 }
 
+.catalog-tile-pf-icon {
```

Comment:
Is this also an upstream bug?  If so, need a link to an upstream issue.

Reply:
I don't believe it's an upstream bug. There is a difference of markup structure between the OperatorHub and Dev Catalog tiles ...  OperatorHub has `<div class="pf-c-card__header"><span><img>` while Dev Catalog has `<div class="pf-c-card__header"><img>`  not sure why... 

- https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx#L386-L407 
- https://github.com/openshift/console/blob/master/frontend/public/components/catalog/catalog-tile.tsx

Reply:
We [pass our own `<img>` to `icon` in order to lazy load](https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx#L386), resulting in the [`span.catalog-tile-pf-icon`](https://github.com/patternfly/patternfly-react/blob/master/packages/react-catalog-view-extension/src/components/CatalogTile/CatalogTile.tsx#L113).

Reply:
These changes (including the one you reordered) should live in `_overrides.scss` since they're overrides/additions to PatternFly rules.

Please add a comment to this rule explaining why it is necessary.

Reply:
On second thought, instead of adding to an existing PatternFly class, you should probably create a new class with a modifier in the classname (e.g., `catalog-tile-pf-icon--aligin-top`) and add the new className in the JSX so the fix is scoped to only where it is needed.

Reply:
Added modifier class.